lambda/parameteTree.spec.ts

```diff
@@ -1,8 +1,10 @@
 import { parameterTree } from './parameterTree.js'
+import { describe, test as it } from 'node:test'
+import { check, objectMatching } from 'tsmatchers'
 
-describe('parameterTree()', () => {
-	it('should create a tree structure from the given SSM paremeters', () =>
-		expect(
+void describe('parameterTree()', () => {
+	void it('should create a tree structure from the given SSM parameters', () => {
+		check(
```

Comment:
[assert.deepEqual](https://nodejs.org/api/assert.html#assertdeepequalactual-expected-message) should be sufficient here, so we don't need tsmatchers at all.

Reply:
Would assert.deepStrictEqual be a better option since deepEqual is marked as legacy in the documentation?

Reply:
Yes, good point.